.travis.yml

```diff
@@ -1,17 +1,20 @@
 language: node_js
 node_js:
 - 4.2.1
-- 6.9.2
 before_install:
 - git config --global user.name "Travis CI Test"
 - git config --global user.email "test@test.com"
 - export docuBranch='master'
 - export NODE_ENV=prod
-- export RESULT_LOC='https://'$DEPLOY_TOKEN'@github.com/yaas/chewie-sample-result.git'
+- export RESULT_LOC='git@github.com:YaaS/chewie-sample-result.git'
```

Comment:
There is special field in .travis.yml to export env variables, `env -> global` I believe